tools/tpm2_createprimary.c

```diff
@@ -178,16 +179,16 @@ int create_primary(TSS2_SYS_CONTEXT *sapi_context) {
     sessionsData.cmdAuthsCount = 1;
     sessionsDataOut.rspAuthsCount = 1;
 
-    inSensitive.t.sensitive.data.t.size = 0;
-    inSensitive.t.size = inSensitive.t.sensitive.userAuth.b.size + 2;
+    ctx.inSensitive.t.size = ctx.inSensitive.t.sensitive.userAuth.b.size + 
+        sizeof(ctx.inSensitive.t.size);
```

Comment:
This requires attribute(packed) on structures to work correctly....

I can't wait for .t and .b to go away....